workflows/theiacov/wf_theiacov_ont.wdl

```diff
@@ -427,5 +427,7 @@ workflow theiacov_ont {
     # QC_Check Results
     String? qc_check = qc_check_task.qc_check
     File? qc_standard = qc_check_task.qc_standard
+    # Non-flu specific outputs
+    String percentage_mapped_reads = select_first([stats_n_coverage.percentage_mapped_reads, flu_track.ha_na_mapped_reads_percentage_string, ""])
```

Comment:
think you accidentally left in a _string here that's causing the CI to fail. 😂 typing what the brain is thinking haha

Reply:
oops ha thanks